free-podcasts-screencasts-es.md

```diff
@@ -43,6 +43,7 @@
 * [La Buhardilla Geek](https://www.ivoox.com/podcast-buhardilla-geek_sq_f1465450_1.html) (podcast)
 * [Maestría JS](https://anchor.fm/maestriajs) (podcast) (Última Actualización, Mayo 2020)
 * [Programar es una Mierda](https://www.programaresunamierda.com) (podcast)
+* [The Dojo MX Livestreams](https://www.youtube.com/playlist?list=PLfeFnTZNTVDMo5ZljjOZu0FquW6f9hqst)(screencast)
```

Comment:
`[Livestreams](playlist link) - The Dojo MX (screencast)`
We try to use the title and other metadata given by the resource.